server/api/scheduler_test.go

```diff
@@ -97,6 +93,35 @@ func (s *testScheduleSuite) TestAPI(c *C) {
 				c.Assert(resp["range-name"], Equals, "test")
 			},
 		},
+		{
+			name:        "evict-leader-scheduler",
+			createdName: "evict-leader-scheduler",
+			args:        []arg{{"store_id", 1}},
+			// Test the scheduler config handler.
+			extraTestFunc: func(name string, c *C) {
+				resp := make(map[string]interface{})
+				listURL := fmt.Sprintf("%s%s%s/%s/list", s.svr.GetAddr(), apiPrefix, server.ScheduleConfigHandlerPath, name)
+				c.Assert(readJSON(listURL, &resp), IsNil)
+				exceptMap := make(map[string]interface{})
+				exceptMap["1"] = []interface{}{map[string]interface{}{"end-key": "", "start-key": ""}}
+				c.Assert(resp["store-id-ranges"], DeepEquals, exceptMap)
+				c.Assert(resp["name"], Equals, "evict-leader-scheduler")
+
+				//using /pd/v1/schedule-config/evict-leader-scheduler/config to add new store to evict-leader-scheduler
+				input := make(map[string]interface{})
+				input["name"] = "evict-leader-scheduler"
+				input["store_id"] = 2
+				updateURL := fmt.Sprintf("%s%s%s/%s/config", s.svr.GetAddr(), apiPrefix, server.ScheduleConfigHandlerPath, name)
+				body, err := json.Marshal(input)
+				c.Assert(err, IsNil)
+				c.Assert(postJSON(updateURL, body), IsNil)
+				resp = make(map[string]interface{})
+				c.Assert(readJSON(listURL, &resp), IsNil)
```

Comment:
check if the `"1"` still exist.

Reply:
@nolouch exceptMap["1"] = []interface{}{map[string]interface{}{"end-key": "", "start-key": ""}}check "1" exist, and exceptMap is not cleared, so after config store "2" to evict-leader-scheduler, the exceptMap has two element "1" and "2", so it can check "1" still exist.

Reply:
got it.